src/redux/actions/projects/createProject.js

```diff
@@ -34,7 +49,44 @@ const createProject = (
   };
 
   try {
-    await dispatch(saveProject(newProjectUuid, newProject));
```

Comment:
So I see you do the saving here, do we use `saveProject` elsewhere in the code? If not you should delete it. And why do you actually do it here?

Reply:
`saveProject` hits a different endpoint than `createProject` (`PUT` and `POST`), you can check in the api PR the differenc ein what these endpoints do

Reply:
And yes, `saveProject` is being used in many places to update the project in the api